snippets/ztp_SriovOperatorConfig.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  configDaemonNodeSelector:
+    "node-role.kubernetes.io/master": ""
+  disableDrain: false
```

Comment:
disableDrain: false should be removed from the example due to the changes in [OCPBUGS-6763](https://issues.redhat.com/browse/OCPBUGS-6763)

Reply:
Updated.